extension/templates/CRM/Banking/Page/Manager.tpl

```diff
@@ -40,6 +40,20 @@ tr.banking-plugin-disabled {
 {else}
 
 {* NORMAL PAGE *}
+<h3>{ts domain='org.project60.banking'}User interface{/ts}</h3>
+<form action="" method="get">
+  <div class="crm-block crm-form-block">
+    <label for="enable_new_ui">{ts domain='org.project60.banking'}User interface{/ts}</label>
+    <select name="enable_new_ui" class="crm-select2">
+      <option value="1" {if $new_ui_enabled}selected="selected"{/if}>{ts domain='org.project60.banking'}New user interface{/ts}</option>
+      <option value="0" {if !$new_ui_enabled}selected="selected"{/if}>{ts domain='org.project60.banking'}Legacy user interface{/ts}</option> 
+    </select>
+    <div class="crm-submit-buttons">
+    <input type="submit" class="crm-form-submit" value="{ts domain='org.project60.banking'}Save{/ts}">
+    </div>
+  </div>
+</form>
+
```

Comment:
Is there a settings page? I have asked Betty where it would make sense to her to have the setting. But if there is a banking settings page that would make sense.

Reply:
see comment below:

> I did move the setting to switch between old and new UI to the settings page, though. I case you hadn't seen it, have a look at the administration console.